python/ray/train/tests/test_tensorflow_trainer.py

```diff
@@ -101,7 +101,7 @@ def train_func():
         else:
             model = build_model()
 
-        model.save("my_model", overwrite=True)
+        model.save("my_model")
```

Comment:
just curious, why did `overwrite` not work.

Reply:
default is having `overwrite=True`.